src/main/java/org/terasology/launcher/gui/javafx/SettingsController.java

```diff
@@ -206,19 +142,14 @@ protected void saveSettingsAction(ActionEvent event) {
         // store changed settings
         try {
             launcherSettings.store();
+            logger.info("Successfully saved current launcher settings");
+            // TODO: Save current state and disable Save button (use MVVM)
         } catch (IOException e) {
             logger.error("The launcher settings can not be stored! '{}'", launcherSettings.getLauncherSettingsFilePath(), e);
             GuiUtils.showErrorMessageDialog(stage, BundleUtils.getLabel("message_error_storeSettings"));
-        } finally {
-            ((Node) event.getSource()).getScene().getWindow().hide();
         }
     }
 
-    @FXML
-    protected void openGameDirectoryAction() {
```

Comment:
Why did you remove this? IMO it's very helpful to be able to open the directories.